.azure-pipelines/solutionValidator.yaml

```diff
@@ -0,0 +1,13 @@
+jobs:
+- job: "SolutionValidations"
```

Comment:
Idea is to expand / use the same pipelines for other solution validation scenarios ? Otherwise just for this test, looks too generic/

Reply:
we are targeting to keep all solution related GitHub Validations here.